posthog/hogql_queries/insights/retention_query_runner.py

```diff
@@ -363,7 +363,7 @@ def to_actors_query(self, interval: Optional[int] = None) -> ast.SelectQuery:
             for i in range(self.query_date_range.total_intervals - interval):
                 retention_query.select.append(
                     ast.Alias(
-                        alias=f"appearance_{i}",
+                        alias=f"{(self.query.retentionFilter.period or 'Day').lower()}_{i}",
```

Comment:
```suggestion
                        alias=f"{(self.query_date_range.interval_name}_{i}",
```

Should be enough. The default is handled there already.